docs/api-markdown-documenter/render-api-documentation.js

```diff
@@ -121,6 +130,53 @@ async function renderApiDocumentation() {
 	console.groupEnd();
 }
 
+/**
+ * Processes, categorizes them based on API item kinds and saves metadata from documents into YAML files.
+ *
+ * The function creates maps from the metadata: one for API items and another for packages.
+ * Results are saved to 'apiData.yaml' and 'packageData.yaml' in the 'data' directory.
+ *
+ * @param {Array<Object>} documents - List of documents containing metadata.
+ * @param {Object} documents[].documentItemMetadata - Metadata for a document item.
+ * @param {string} documents[].documentItemMetadata.apiItemName - Name of the API item.
+ * @param {string} documents[].documentItemMetadata.apiItemKind - Kind of the API item (e.g., Class, Interface, Package).
+ * @param {string} documents[].documentItemMetadata.packageName - Name of the package to which the API item belongs.
+ *
+ * @returns {void}
+ */
+function processMetaDataFromDocuments(documents) {
+	const { APIMap, packageMap } = documents.reduce(
+		(
+			{ APIMap, packageMap },
+			{ documentItemMetadata: { apiItemName, apiItemKind, packageName } },
+		) => {
+			if (apiItemKind === ApiItemKind.Package) {
+				return {
+					APIMap,
+					packageMap: { ...packageMap, [apiItemName]: packageName },
+				};
+			}
+
+			if (
+				[ApiItemKind.Class, ApiItemKind.Interface, ApiItemKind.Enum].includes(apiItemKind)
```

Comment:
Do we need `namespace` here as well?

Reply:
No we do not, we are not exposing namespace in the nav bar, but that begs the question do we need it in getApiItemTransformationConfigurationWithDefaults.documentBoundaries on line 52?

Reply:
Okay, just wanted to double check. 

I would argue we still probably want namespaces included in `documentBoundaries` to avoid too much noise in the root package document. Namespace exports are generally pretty large, and are semantically intended to introduce scoping to the API, so pulling them out as separate docs makes the most sense to me.

As for whether or not we add them to the nav - I personally think it would be useful to do. If a package is exporting a namespace (which includes anything of the form `export * as Foo from ...`), we want that to be easily discoverable.